packages/mux-player/src/template.ts

```diff
@@ -34,62 +33,46 @@ export const template = (props: MuxTemplateProps) => html`
       slot="media"
       crossorigin
       playsinline
-      ${props.autoplay ||
+      autoplay="${props.autoplay ||
       props.streamType === StreamTypes.LIVE ||
-      props.streamType === StreamTypes.LL_LIVE
-        ? "autoplay"
-        : ""}
-      ${props.muted ? "muted" : ""}
-      ${props.debug ? "debug" : ""}
-      ${props.preferMse ? "prefer-mse" : ""}
-      ${props.startTime != null ? `start-time="${props.startTime}"` : ""}
-      ${props.poster
-        ? `poster="${
-            props.poster ?? getPosterURLFromPlaybackId(props.playbackId)
-          }"`
-        : ""}
-      ${props.playbackId ? `playback-id="${props.playbackId}"` : ""}
-      ${props.envKey ? `env-key="${props.envKey}"` : ""}
-      ${props.streamType ? `stream-type="${props.streamType}"` : ""}
-      ${props.metadata?.video_id
-        ? `metadata-video-id="${props.metadata.video_id}"`
-        : ""}
-      ${props.metadata?.video_title
-        ? `metadata-video-title="${props.metadata.video_title}"`
-        : ""}
-      ${props.metadata?.viewer_user_id
-        ? `metadata-viewer-user-id="${props.metadata.viewer_user_id}"`
-        : ""}
+      props.streamType === StreamTypes.LL_LIVE}"
+      muted="${props.muted}"
+      debug="${props.debug}"
+      prefer-mse="${props.preferMse}"
+      start-time="${!!props.startTime ? props.startTime : false}"
+      poster="${!!props.poster
+        ? getPosterURLFromPlaybackId(props.playbackId)
+        : false}"
+      playback-id="${props.playbackId ?? false}"
+      env-key="${props.envKey ?? false}"
+      stream-type="${props.streamType ?? false}"
+      metadata-video-id="${props.metadata?.video_id ?? false}"
+      metadata-video-title="${props.metadata?.video_title ?? false}"
+      metadata-viewer-user-id="${props.metadata?.viewer_user_id ?? false}"
```

Comment:
simpler attribute syntax, only thing is that you have to remember to add `false` to remove an attribute for string attributes.